validator/package.json

```diff
@@ -15,8 +15,8 @@
     "preinstall": "node ../build-system/check-package-manager.js"
   },
   "devDependencies": {
-    "google-closure-compiler": "20180506.0.0",
-    "google-closure-library": "20180506.0.0",
-    "jasmine": "3.1.0"
+    "google-closure-compiler": "20190709.0.0",
+    "google-closure-library": "20190709.0.0",
+    "jasmine": "3.4.0"
```

Comment:
If you add `validator/package.json ` to `renovate.json` (see below), you can benefit from semi-automatic updates when new versions of these packages are released.

https://github.com/ampproject/amphtml/blob/13baabce3d2cf2b92025872033640c2a93716b25/renovate.json#L15-L19

Reply:
I plan to do that in a separate PR. Maybe that's unnecessary. Thanks for the pointer!